src/api/app/components/sourcediff_tab_component.rb

```diff
@@ -1,5 +1,7 @@
 # frozen_string_literal: true
 
+require 'action_sourcediff_parser'
```

Comment:
Why is this `require` statement needed here, but not in the other view component above?

Reply:
I forgot to remove this `require` when I moved the code from `src/api/lib/` to `src/api/services/`. Removed. Thanks!